dan_layer/core/src/models/vote_message.rs

```diff
@@ -159,4 +165,25 @@ impl VoteMessage {
     pub fn all_shard_pledges(&self) -> &ShardPledgeCollection {
         &self.all_shard_pledges
     }
+
+    pub fn merkle_proof(&self) -> Option<BalancedBinaryMerkleProof<ValidatorNodeBmtHasherBlake256>> {
+        self.merkle_proof.clone()
+    }
+
+    // TODO: impl CBOR for merged merkle proof
```

Comment:
It was a leftover comment, deleted, thanks.

Reply:
Thanks - I meant change the encoding to use CBOR instead of bincode because we dont use this anywhere else and it adds a dependency. But CBOR will result in a larger encoding so let's leave to and we can decide how we want to do this later